src/index.js

```diff
@@ -25,10 +25,10 @@ function writeLog(entries, logger) {
     });
 }
 
-function clone(deps: Array<string>, logger) {
+function clone(deps: Array<string>, logger, token: string) {
     resolver
         .on('invalid', (config: ZelConfig) => logger.error(LOG.INVALID, config.repoName))
-        .validate(deps)
+        .validate(deps, { token })
```

Comment:
Instead of passing the `token` here to `validate()`, we may want to instantiate the `GitHubResolver` with the provided `token` instead since other future resolvers may not require a token. Same goes for fetchers.